### PR TITLE
 VSOCK implementation for LibIIO

### PR DESCRIPTION
**Pull request by vilasrk**
Tracked-On:
Signed-off-by: vilasrk <vilas.r.k@intel.com>

### CODE REVIEW

**Review by JaikrishnaNemallapudi — APPROVED** [review]
LGTM

File: sensors/mediation/sensor_hal_default.te [hunk 1]
```diff
@@ -4,8 +4,10 @@ allow hal_sensors_default self:socket create_socket_perms;
 allowxperm hal_sensors_default self:socket ioctl unpriv_sock_ioctls;
 allow hal_sensors_default serial_device:chr_file rw_file_perms;
 
-allow hal_sensors_default self:tcp_socket { create read write connect name_connect getopt setopt };
-dontaudit hal_sensors_default default_prop:file { open read getattr map };
-allow hal_sensors_default port:tcp_socket { name_connect };
+#allow hal_sensors_default self:tcp_socket { create read write connect name_connect getopt setopt };
+#dontaudit hal_sensors_default default_prop:file { open read getattr map };
```
Comment (ceiba1985) [inline sensors/mediation/sensor_hal_default.te]: can you delete these comment sentences?

Comment (vilasrk) [reply sensors/mediation/sensor_hal_default.te]: done

File: sensors/mediation/sensor_hal_default.te [hunk 2]
```diff
@@ -4,8 +4,4 @@ allow hal_sensors_default self:socket create_socket_perms;
 allowxperm hal_sensors_default self:socket ioctl unpriv_sock_ioctls;
 allow hal_sensors_default serial_device:chr_file rw_file_perms;
```
Comment (rajucm2007) [inline sensors/mediation/sensor_hal_default.te:6]: check, may be not require

**Review by rajucm2007 — CHANGES_REQUESTED** [review]

**Review by rajucm2007 — APPROVED** [review]
Looks good to me

**Review by JaikrishnaNemallapudi — APPROVED** [review]
LGTM

**Review by ceiba1985 — APPROVED** [review]

**Pull request closed without merging**